eng/update-dependencies/Options.cs

```diff
@@ -31,96 +35,44 @@ public class Options
 
         public string? GitHubUser { get; private set; }
 
-        public string NuGetInfoPath { get; private set; } = "eng/nuget-info.json";
+        public string NuGetInfoPath { get; private set; } = DefaultNuGetInfoPath;
```

Comment:
Why is it necessary to specify a default for the property here as well as the GetCliSymbols method?

Reply:
I guess it's not needed.